pkg/cmd/server/bootstrappolicy/old_policy_test.go

```diff
@@ -0,0 +1,897 @@
+package bootstrappolicy_test
+
+import (
+	"testing"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/apis/autoscaling"
+	"k8s.io/kubernetes/pkg/apis/batch"
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/util/sets"
+
+	"github.com/openshift/origin/pkg/api"
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	authorizationapiv1 "github.com/openshift/origin/pkg/authorization/api/v1"
+	"github.com/openshift/origin/pkg/authorization/rulevalidation"
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	imageapi "github.com/openshift/origin/pkg/image/api"
+	projectapi "github.com/openshift/origin/pkg/project/api"
+	routeapi "github.com/openshift/origin/pkg/route/api"
+)
+
+func TestClusterRoles(t *testing.T) {
+	currentRoles := bootstrappolicy.GetBootstrapClusterRoles()
+	oldRoles := oldGetBootstrapClusterRoles()
+
+	// old roles don't have the SAs appended, so run through them.  The SAs haven't been converted yet
+	for i := range oldRoles {
+		oldRole := oldRoles[i]
+		newRole := currentRoles[i]
+
```

Comment:
> make sure the role names match

Ok, but just in case one role happens to be identical to another one?  You understand this is a "for one commit" test, right?